app/models/appointment.rb

```diff
@@ -17,8 +16,16 @@ class Appointment < ApplicationRecord
     end
   end
 
+  validate :client_is_unchanged, on: [:update]
```

Comment:
Hmm...nvm...didn't notice you were adding errors rather than returning a boolean.